tools.html

```diff
@@ -2,7 +2,7 @@
 layout: general
 highlight_section: tools
 title: Get tools
-description: Prepare your environment with DevAssistant. Collaborate with Docker and Vagrant.
+description: Prepare your environment with DevAssistant. Collaborate with Docker, Vagrant and Virt-builder.
```

Comment:
Not sure about expanding this description further. We probably don't want *all* the tooks mentioned.